rust/operator-binary/src/controller.rs

```diff
@@ -931,7 +942,7 @@ async fn create_shared_internal_secret(trino: &TrinoCluster, client: &Client) ->
         .is_none()
     {
         client
-            .apply_patch(FIELD_MANAGER_SCOPE, &secret, &secret)
+            .apply_patch(CONTROLLER_NAME, &secret, &secret)
```

Comment:
I would think that it makes sense to include the shared secret as well. I does not need to stay persistent between Trino Clusters as far as i know. CC @maltesander

Reply:
Yeah each cluster creates a shared secret for internal communication. So they are independent.

Reply:
Nevermind, they have a owner reference